frontend/src/pages/aboutPage.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import '../styles/aboutPage.css';
 import brandonImage from '../assets/BArsenault.jpg';
 import davidImage from '../assets/DClaphan.jpg';
-import AboutModal from '../components/aboutModal';
+import AboutModal from '../components/AboutModal';
```

Comment:
I am getting an error from this I think it needs to be lowercase "a" in '/aboutModal'

Reply:
It does. I tried to change the filename to be inline with our naming conventions but for some reason that change wouldn't push.

Reply:
fixed!